frontend/src/views/GNewShoot.vue

```diff
@@ -7,158 +7,163 @@ SPDX-License-Identifier: Apache-2.0
 <template>
   <div
     v-if="sortedInfrastructureKindList.length"
-    class="newshoot-container"
+    class="d-flex flex-column fill-height"
   >
-    <v-container
-      fluid
-      class="newshoot-cards"
+    <div
```

Comment:
I would propose the following layout:
```vue
  <div
    v-if="sortedInfrastructureKindList.length"
    class="d-flex flex-column fill-height"
  >
    <v-container
      class="overflow-auto"
      fluid
    >
    ...
    </v-container>
    <div>
      <g-message
        v-if="errorMessage"
        v-model:message="errorMessage"
        v-model:detailed-message="detailedErrorMessage"
        color="error"
        class="ma-0"
        tile
      />
      <v-divider />
      <div class="d-flex align-center justify-end toolbar">
      ...
      </div>
    </div>
  </div>
```
* css class `newshoot-cards` is replaced by `overflow-auto`
* remove unnecesaary div's, classes (cardTitle, newshoot-cards) and refs